include/nbl/ext/FFT/FFT.h

```diff
@@ -9,24 +9,37 @@
 #include "nbl/video/IGPUShader.h"
 #include "nbl/asset/ICPUShader.h"
 
+
 namespace nbl
 {
 namespace ext
 {
 namespace FFT
 {
 
+typedef uint32_t uint;
+struct uvec3 {
+	uint x,y,z;
+};
+struct uvec4 {
+	uint x,y,z,w;
+};
+#include "nbl/builtin/glsl/ext/FFT/parameters.glsl";
+
 class FFT : public core::TotalInterface
 {
 	public:
+		struct Parameters_t : nbl_glsl_ext_FFT_Parameters_t {
```

Comment:
because of alignment to 16 bytes, I'd add a `uint padding` member to the GLSL struct.